client/pk-console.c

```diff
@@ -637,30 +637,21 @@ pk_console_progress_cb (PkProgress *progress, PkProgressType type, gpointer data
 }
 
 static void
-pk_console_finished_cb (GObject *object, GAsyncResult *res, gpointer data)
+pk_console_process_results (PkResults *results, PkConsoleCtx *ctx, GError *err)
```

Comment:
nit: Just to make abundantly clear that this is not a return error and a specific error is expected, naming the parameter something like `finish_error` would maybe make sense, so it's easier to grasp for someone reading this in future.

Reply:
Sure. Making it `const` will be an option too.